packages/jest-config-react-native/lib/index.js

```diff
@@ -15,16 +15,13 @@ module.exports = (options) => {
 
     return (config) => ({
         ...config,
+        testEnvironment: reactNativePreset.testEnvironment,
         haste: {
             ...config.haste,
             ...reactNativePreset.haste,
         },
         transform: {
-            ...config.transform,
-            '\\.(png|jpg|jpeg|gif|webp|svg|ico|bmp)$': require.resolve('../../transforms/file'),
-            '\\.(eot|ttf|woff|woff2|otf)$': require.resolve('../../transforms/file'),
-            '\\.(mp3|flac|wav|aac|ogg|oga|mp4|m4a|webm|ogv)$': require.resolve('../../transforms/file'),
-            '\\.(obj|mtl|fnt|glb)$': require.resolve('../../transforms/file'),
+            ...reactNativePreset.transform,
```

Comment:
I think we can still use our own transforms because they work just fine! I don't exactly remember the reasoning behind using them and not using React Native's transform. Should we copy our transforms from the web config to the React Native config or dedup them somehow?

Reply:
The change was intentional because I wasn't 100% sure of the answer to the question above.

Reply:
We can put them in the `base` package just like we are doing in eslint config base in regards to the test patterns.

Alternatively, yes we can dedup, but meh :P

Reply:
Sounds reasonable! Will do just that.

Reply:
Done ✅